shell/src/main/java/alluxio/cli/fs/command/DistributedCpCommand.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.cli.CommandUtils;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.job.JobGrpcClientUtils;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.job.migrate.MigrateConfig;
+
+import org.apache.commons.cli.CommandLine;
+
+import javax.annotation.concurrent.ThreadSafe;
+import java.io.IOException;
```

Comment:
This should be in its own import group above javax

Reply:
done